### DISCUSSION TITLE
borderlayout

### DISCUSSION THREAD

**Discussion by Petricevic** [fyne-io/fyne#3502] [Q&A]
Good morning everyone,
it happened to me that with nil objects it went wrong.
I modified line: 51 and line 70 adding:
`
  		if child == nil {
			continue
		} 

  `

**Comment by Jacalz** [comment]
Please open a bug report if you have noticed a bug instead of a discussion. Or better yet, you can open a PR to fix it :)

**Reply by Jacalz** [reply]
It obviously shouldn't crash but it also is not quite ideal to pass nil objects. I'd avoid that if possible.

**Reply by andydotxyz** [reply]
With border it is quite normal to pass `nil` for those edges you don't want to fill

**Reply by Jacalz** [reply]
Indeed. I presumed that "it went wrong" implied a crash (for non edge items), but maybe not?
